templates/zerver/help/send-a-group-of-people-a-private-message.md

```diff
@@ -1,6 +1,6 @@
 # Private Group Messages
 ## There are three ways to send a private message to a group of people:
```

Comment:
Remember my styling comments in your other [PR](https://github.com/zulip/zulip/pull/3162)? Can you please update this doc with the changes I suggested in that PR? Most of the same styling errors are here and I don't want to repeat them.